pkg/devserver/devserver.go

```diff
@@ -172,8 +177,18 @@ func start(ctx context.Context, opts StartOpts) error {
 	if opts.PostgresURI != "" {
 		dbDriver = "postgres"
 	}
-	dbcqrs := base_cqrs.NewCQRS(db, dbDriver)
-	hd := base_cqrs.NewHistoryDriver(db, dbDriver)
+	dbcqrs := base_cqrs.NewCQRS(db, dbDriver, sqlc_postgres.NewNormalizedOpts{
+		MaxIdleConns:    opts.PostgresMaxIdleConns,
+		MaxOpenConns:    opts.PostgresMaxOpenConns,
+		ConnMaxIdle:     opts.PostgresConnMaxIdleTime,
+		ConnMaxLifetime: opts.PostgresConnMaxLifetime,
+	})
+	hd := base_cqrs.NewHistoryDriver(db, dbDriver, sqlc_postgres.NewNormalizedOpts{
+		MaxIdleConns:    opts.PostgresMaxIdleConns,
+		MaxOpenConns:    opts.PostgresMaxOpenConns,
+		ConnMaxIdle:     opts.PostgresConnMaxIdleTime,
+		ConnMaxLifetime: opts.PostgresConnMaxLifetime,
+	})
```

Comment:
nit: looks like you could extract the normalized opts struct to a variable and reuse it in both constructors